alerts/couchbase/unrecoverable-oom-errors.json

```diff
@@ -0,0 +1,36 @@
+{
+    "displayName": "Couchbase - Unrecoverable OOM Errors",
+    "documentation": {
+        "content": "If `bucket.error.oom.count` that are `unrecoverable` increase above 0 should be a cause for concern because it implies the couchbase server is either underprovisioned or an application is over-utilizing the couchbase server.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
+    "conditions": [
+        {
+            "displayName": "VM Instance - workload/couchbase.bucket.oom.count",
+            "conditionThreshold": {
+                "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/couchbase.bucket.oom.count\" AND metric.labels.error_type = \"unrecoverable\"",
+                "aggregations": [
+                    {
+                        "alignmentPeriod": "300s",
+                        "crossSeriesReducer": "REDUCE_NONE",
+                        "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                ],
+                "comparison": "COMPARISON_GT",
+                "duration": "30s",
```

Comment:
condition_threshold.duration had an invalid value of \"30s\": only multiples of 1 minute are supported